common/changes/office-ui-fabric-react/revert-react-portals_2018-05-10-22-27.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Revert change to use React.createPortal, until focus leaking issues can be resolved",
```

Comment:
generalize this comment to be "... until **event** leaking issues can be resolved"